README.md

```diff
@@ -29,6 +34,37 @@ pip install -e ".[fms-accel]"
 ```
 `fms-acceleration` is a collection of plugins that packages that accelerate fine-tuning / training of large models, as part of the `fms-hf-tuning` suite. For more details on see [this section below](#fms-acceleration).
 
+
+### Using versions of python dependencies that are known to be compatible with fms-hf-tuning
+
+This repository uses [`poetry`](https://python-poetry.org/) for reproducible builds. You can use poetry to create the same virtual environment that we use for our CI/CD tests like so:
+
+```
+python -m venv venv
+. ./venv/bin/activate
+pip install poetry
+
+poetry install
+```
+
+> Note: After installing, if you wish to use [FlashAttention](https://github.com/Dao-AILab/flash-attention), then you need to install these requirements:
+ 
+```
+poetry install --with dev,flash-attn
+```
+
+If you wish to use [aim](https://github.com/aimhubio/aim), then you need to install it:
+```
+poetry install --with aim
+```
+
+If you wish to use [fms-acceleration](https://github.com/foundation-model-stack/fms-acceleration), you need to install it. 
+```
+poetry install --with fms-accel
+```
+`fms-acceleration` is a collection of plugins that packages that accelerate fine-tuning / training of large models, as part of the `fms-hf-tuning` suite. For more details on see [this section below](#fms-acceleration).
```

Comment:
In order to make the the README.md cleaner, should we move this section to the [Development](https://github.com/foundation-model-stack/fms-hf-tuning/blob/main/CONTRIBUTING.md#development) in CONTRIBUTING.md? Replacing the current Installation.